common/lib/plugins/failover2/failover2_plugin.ts

```diff
@@ -495,7 +496,7 @@ export class Failover2Plugin extends AbstractConnectionPlugin implements CanRele
   async releaseResources(): Promise<void> {
     const hostListProvider: HostListProvider = this.pluginService.getHostListProvider();
     if (this.pluginService.isBlockingHostListProvider(hostListProvider)) {
-      await hostListProvider.clearAll();
+      RdsHostListProvider.clearAll();
```

Comment:
The monitoring host list provider's clearAll method also calls RdsHostListProvider.clearAll(). With this change, we're skipping the `await MonitoringRdsHostListProvider.monitors.clear()` call. Do we need that?